### PR TITLE
Updating Organiser Page text

### PR DESCRIPTION
**Pull request by JiggyPete**
This PR is intended to give a clearer picture of what is expected of organisers.

Emphasising:
* 1 workshop per city (work together folks)
* Representation Matters in each Workshop Team
* It is up to Workshop Teams to promote their workshop

Screenshot 2019-08-27 at 18 23 17

### CODE REVIEW

File: app/views/organise/index.html.erb [hunk 1]
```diff
@@ -8,22 +8,47 @@
...
+    </li>
+    <li>
+      A venue with a projector and audio
+    </li>
+    <li>
+      An awesome team 🎉
+      <br/>
+      Your team will include a facilitator to introduce the sections of the workshop and mentors (around one mentor for every five attendees) who can advise attendees with their experience of applying to CFPs in the past.
+      <br/>
+      Representation Matters. When putting together your workshop team, the people involved need to reflect the people we are encouraging to attend.
```
Comment (JiggyPete) [inline app/views/organise/index.html.erb]: Representation Matters

So does transparency. 

Invite **all** team members to the site once you have a workshop setup, it'll encourage more potential attendees to participate.

File: app/views/organise/index.html.erb [hunk 2]
```diff
@@ -8,22 +8,47 @@
...
+    </li>
+    <li>
+      Ensure you agree with our <%= link_to "Code of Conduct", code_of_conduct_path %> as you will be in charge of enforcing it. Mandatory training will be provided.
```
Comment (Chimney42) [inline app/views/organise/index.html.erb]: do we want to add something here that an event cannot move forward if not all of the mentors & facilitator attended that training?

File: app/views/organise/index.html.erb [hunk 3]
```diff
@@ -8,22 +8,47 @@
...
+    </li>
+    <li>
+      A ticketing page
```
Comment (Chimney42) [inline app/views/organise/index.html.erb]: maybe add a recommendation like Tito

File: app/views/organise/index.html.erb (hunk 2 above)
Comment (JiggyPete) [reply app/views/organise/index.html.erb]: Great point, thanks!

File: app/views/organise/index.html.erb (hunk 3 above)
Comment (JiggyPete) [reply app/views/organise/index.html.erb]: 👍 Think we do that when people are filling in the details however it doesn't hurt to add here too.

Hopefully getting this issue built means we can automate a standard ticket page for each workshop https://github.com/JiggyPete/global-diversity-cfp-day-site/issues/24

**Comment by JiggyPete** [comment]
Thanks for the feedback @Chimney42

**Pull request merged**